nengo_loihi/simulator.py

```diff
@@ -417,6 +417,7 @@ def run_steps(self, steps):
                     self.sims["emulator"].step()
                     self.handle_chip2host_communications()
             else:
+                raise Exception
```

Comment:
What's this exception for? Same down below.

Reply:
Ahh, because these never get hit. In the future, I would put that in the exception comment. These get removed by a later commit, though, so not worth fixing here.